frontend/src/api/Api.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+
+const baseUri = 'http://localhost:61346/api';
+
+const Api = axios.create({
+  baseURL: baseUri,
+});
+
+Api.interceptors.response.use(
+  (response) => response,
+  (error) => {
+    if (error.response.status === 401) {
```

Comment:
Are you sure that we do not return unauthorized in backend when token is still not expired?

Reply:
Like I don't know, in cases where some ids do not match, even though the token is valid

Reply:
By design Unauthorized only gets called if the token is expired or the user did some fuckery with the storage

Reply:
Not sure if your worry is valid. It'd be good to avoid, because we don't have a refresh token yet (imma implement it now), when the token expires the user is left hanging. We could check if the response gets a 401 everywhere but that's lame. Interceptors are cool